pkgs/default.nix

```
@@ -40,6 +40,7 @@ in
 
   # official servers
   context7-mcp = pkgs.callPackage ./official/context7 { };
+  mcp-tavily = pkgs.callPackage ./official/tavily { };
```

```suggestion
  tavily-mcp = pkgs.callPackage ./official/tavily { };
```
The package name should match the official one.
Confusingly, some third-party MCP servers are also called `mcp-tavily`, which might be a source of this conflict.
https://pypi.org/project/mcp-tavily/
https://www.npmjs.com/package/@mcptools/mcp-tavily